admin/billing/noop.go

```diff
@@ -70,6 +70,10 @@ func (n noop) CancelSubscriptionsForCustomer(ctx context.Context, customerID str
 	return nil
 }
 
+func (n noop) FindSubscriptionsPastTrialPeriod(ctx context.Context) ([]*Subscription, error) {
+	return []*Subscription{{Customer: &Customer{}, Plan: &Plan{Quotas: Quotas{}}}}, nil
+}
```

Comment:
Can it just return an empty list? Might be unexpected to get a value with nil fields